migrate.go

```diff
@@ -244,7 +244,7 @@ func selectMigrationsTableSql(driver string) string {
 	case "mysql":
 		return "DESC migrations"
 	case "postgres":
-		return "SELECT * FROM migrations ORDER BY id_migration;"
+		return "SELECT * FROM migrations WHERE false ORDER BY id_migration;"
```

Comment:
hmm? can you run `bee migrate` without error on Postgres? I cannot, if `migrations` table has data, your query would return data instead of column names, that's why I need to add `WHERE false` so it returns 0 record. You can try run `bee migrate` the second time, it would fail.